adapter/src/main/java/com/twilio/kudu/sql/schema/KuduSchema.java

```diff
@@ -258,14 +261,34 @@ protected Map<String, Table> getTableMap() {
           }
           queryBuilder.append(Util.toString(groupByExprs, "", ", ", ""));
 
+          // TODO figure out why adding the ORDER BY leads to the view not being chosen
```

Comment:
nit: remove commented lines

Reply:
Ideally while creating the SQL statement that specified the materialized view should also have the ORDER BY clause, but for some reason when I add this the view isn't chosen for queries. Currently, the view sql only has the GROUP BY clause and the KuduSortRule checks the sort order of the query vs the view and is matched. 
I left this code in with the TODO to look into this.